graphics/ZBar/Portfile

```
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+name                    ZBar
+github.setup            mchehab zbar 0.23.90
```

```suggestion
github.setup            mchehab zbar 0.23.90
```

From @reneeotten https://github.com/macports/macports-ports/pull/6344#discussion_r377392262:

> most, but not all, of the port names are in lower case. From last years GSoC where we started working on a MacPorts backend for upt it became clear that a standardized naming conventions is important. Admittedly, we didn't reach a broad consensus about it, but using lowercase as this is customary seems reasonable - so that's what I would prefer here.

I think that should also be applicable here. We can then remove `name` since `github.setup` deals with it for us.